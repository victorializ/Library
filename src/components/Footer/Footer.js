import React from 'react';

import { Segment } from 'semantic-ui-react'

import { constants  } from '../../i18n';

function Footer() {
  return (
    <Segment textAlign='center'>{constants.copyright}</Segment>
  );
}

export { Footer };