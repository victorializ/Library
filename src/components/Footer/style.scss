.app__footer {
    height: 5vh;
}