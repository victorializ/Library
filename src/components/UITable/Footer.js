import React from 'react';
import { Segment } from 'semantic-ui-react'

import { constants } from '../../i18n';

function Footer() {
  return (
    <footer>
      <Segment id="footer" className="app__footer">
        {constants.copyright}
      </Segment>
    </footer>
  );
}

export { Footer };