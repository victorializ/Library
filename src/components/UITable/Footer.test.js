import React from 'react';
import { shallow } from 'enzyme';

import { Footer } from './Footer';

describe('Footer component', () => {
    it('renders correctly', () => {
        expect(shallow(<Footer />)).toMatchSnapshot();
    });   
});