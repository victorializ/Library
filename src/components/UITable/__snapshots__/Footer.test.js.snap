// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<footer>
  <Segment
    className="app__footer"
    id="footer"
  >
    @2019 Author
  </Segment>
</footer>
`;
